### ISSUE TITLE
OIPS-24 Instead of buying so much fruit, lets buy some veggies too

### ISSUE THREAD

**Issue by renaynay** [ethdevberlin/OIPS#24]
Ligi and I were discussing that a bunch of fruits get leftover weekly and end up going to waste. Apples hold for a couple weeks, but bananas don't. Also I feel like all the snacks we have in the office (besides maybe nuts) are sweet. I think we should buy a couple veggies in the beginning of the week and chop them up into snack-size then place them in a Tupperware for anyone who wants to snack on something that's not sugary.

We could buy: 
* bell peppers
* baby tomatoes
* carrots
* celery

(essentially things that can stay for a couple days, cut, without going bad)

We can also maybe make a yogurt-based dip to go along with it. I will look up some recipes.

**Comment by ligi** [comment]
:+1:  This might also help to finally close #4 and also go towards #12 - maybe we can leverage #13 
And one more thing about the bananas: not only get they bad very fast (have seen quite a couple getting bad here) - they also have a big footprint as they are not local in contrast to apples, tomatoes, carrots, ..
Really hope we can get this implemented - maybe also use Kraut&Rübe as a supplier - @mvglasenapp you where writing here: https://github.com/ethdevberlin/OIPS/issues/13#issuecomment-563913833 that you used them - did you continue? Or was there a problem?
And sure there can also be bananas between - we should not be dogmatic - but it should be so few that there is no chance they will get bad.

**Comment by mvglasenapp** [comment]
Yep, the new groceries team started purchasing fruit at Kraut&Ruebe a while ago, but they seem to have started again taking them from Lidl. I will make sure that they take fruit & vegetables from Kraut&Rube again from next week. I will also limit the amount of bananas in order to have them consumed at the end of the week and we start with vegetables from this week (only cherry tomatoes, carrots & 2 avocados for now). We will try out other kinds of vegetables next week (selery...). So according to their "success", I will figure out the needed quantities and continue to order them. Please let me know on the black board when you want to try out other vegetables

**Comment by franzihei** [comment]
Cucumbers. 💚 But I'll be on holiday for two weeks (starting from next week), so maybe not now, if nobody else likes them.
